subprojects/language-native/language-native.gradle.kts

```diff
@@ -89,3 +89,7 @@ gradlebuildJava {
 classycle {
     excludePatterns.set(listOf("org/gradle/language/nativeplatform/internal/**"))
 }
+
+tasks.named<Test>("integMultiVersionTest").configure {
+    include("org/gradle/language/assembler/AssemblyLanguageIncrementalBuildIntegrationTest.class")
+}
```

Comment:
❓ Why do we need to list this explicitly?

Reply:
We don't need it. I left it for testing something on CI, looks like I forgot to remove it. Reverted.